plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/storage/IscsiAdmStorageAdaptor.java

```diff
@@ -331,12 +331,6 @@ public boolean disconnectPhysicalDisk(String volumeUuid, KVMStoragePool pool) {
 
     @Override
     public boolean disconnectPhysicalDisk(Map<String, String> volumeToDisconnect) {
```

Comment:
This code is not needed as the PowerFlex/ScaleIO adaptor method is called now, during the StopCommand.